src/controllers/storeController.ts

```diff
@@ -59,8 +61,44 @@ async function destroyStore(req: Request, res: Response, next: NextFunction) {
   }
 }
 
+async function retrieveHawkerCentre(req: Request, res: Response, next: NextFunction) {
```

Comment:
Can just `retrieveHawkerCentre` in `retrieveStore`.

Reply:
Should be able to retrieve address with store using `include`, e.g. 
`const store = await Store.findOne({ include: HawkerCentre, id: id  });`.

Reply:
👍